pkg/controller/configuration/ensure.go

```diff
@@ -701,12 +701,17 @@ func (c *Controller) ensureTerraformPlan(configuration *terraformv1alpha1.Config
 
 		// @step: lets build the options to render the job
 		options := jobs.Options{
-			AdditionalJobSecrets: state.additionalJobSecrets,
+			AdditionalAnnotations: state.provider.GetAnnotations(),
```

Comment:
should this be JobAnnotations?

Reply:
Updated the job option to make it more clean what it's actually setting ..

Reply:
I mean `state.provider.JobAnnotations()` rather than `state.provider.GetAnnotations()` - functional, not a style comment :)